flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/functions/SqlDateTimeUtils.java

```diff
@@ -253,7 +254,13 @@ public static SqlTimestamp toSqlTimestamp(String dateStr, String format) {
 				return SqlTimestamp.fromLocalDateTime(ldt);
 			}
 		} catch (DateTimeParseException e) {
-			return null;
+			// fall back to support cases like '1999-9-10 05:20:10'
+			try {
+				Timestamp ts = Timestamp.valueOf(dateStr);
```

Comment:
My concern is using try catch is not performance-friendly.

Reply:
So what's your opinion about the exceptions? Just throw them out (like calcite)?

Reply:
I'm OK with `try...catch` if there is no better solution because:
- It is a corner case, should not be normal.
- Cast string to timestamp is a slow operation previous. `try...catch` should not be a big performance impact on it.

Reply:
I'm fine with current approach then.